gradle-baseline-java/src/test/groovy/com/palantir/baseline/BaselineJavaVersionIntegrationTest.groovy

```diff
@@ -343,14 +343,14 @@ class BaselineJavaVersionIntegrationTest extends IntegrationSpec {
         gradleVersionNumber << GRADLE_TEST_VERSIONS
     }
 
-    def '#gradleVersionNumber: jdkToolchainsAutoManagement only sets the language versions'() {
+    def '#gradleVersionNumber: when setupJdkToolchains=false no toolchains are configured by gradle-baseline'() {
         when:
         // language=gradle
         buildFile << '''
         javaVersions {
             libraryTarget = 11
             runtime = 21
-            jdkToolchainsAutoManagement = true
+            setupJdkToolchains = true
```

Comment:
shouldn't this be false?

Reply:
good spot, yes, it should. I realized that these test actually use the auto-detected jdks instead of using the `jdk-latest` plugin, which make my test successful regardless of whether the flag is set or not. Fixing it now.